silverstrike/templates/silverstrike/base.html

```diff
@@ -20,162 +20,177 @@
     <link rel="stylesheet" href="{% static 'vendor/css/skin-blue-light.min.css' %}">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,300italic,400italic,600italic">
 
+    {#    For Dates#}
+    <link rel="stylesheet" href="{% static 'silverstrike/css/daterangepicker.css' %}">
+
 
     <title>SilverStrike</title>
-  </head>
-  <body class="hold-transition skin-blue-light sidebar-mini">
-    <div class="wrapper">
+</head>
+<body class="hold-transition skin-blue-light sidebar-mini">
+<div class="wrapper">
 
-      <header class="main-header">
+    <header class="main-header">
         <a href="/" class="logo">
-          <!-- mini logo for sidebar mini 50x50 pixels -->
-          <span class="logo-mini"><b>AgS</b></span>
-          <!-- logo for regular state and mobile devices -->
-          <span class="logo-lg"><b>Silver</b>strike</span>
+            <!-- mini logo for sidebar mini 50x50 pixels -->
+            <span class="logo-mini"><b>AgS</b></span>
+            <!-- logo for regular state and mobile devices -->
+            <span class="logo-lg"><b>Silver</b>strike</span>
         </a>
         <nav class="navbar navbar-static-top">
+<<<<<<< HEAD
```

Comment:
I know I said you should revert this file, here is a part from a merge that got through.